.env.production.template

```
@@ -162,3 +162,17 @@ NEXT_TELEMETRY_DISABLED=1
+
+# -----------------------------------------------
+# 12. THRESHOLD CONFIGURATION
+# -----------------------------------------------
+NEXT_PUBLIC_INNOVATORS_THRESHOLD=3
+NEXT_PUBLIC_COLLABORATORS_THRESHOLD=3
+NEXT_PUBLIC_FAQ_THRESHOLD=1
+
+# -----------------------------------------------
+# 12. THRESHOLD CONFIGURATION
+# -----------------------------------------------
+NEXT_PUBLIC_INNOVATORS_THRESHOLD=3
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The NEXT_PUBLIC_INNOVATORS_THRESHOLD key is duplicated

(DuplicatedKey)